apiserver/facades/client/client/status.go

```diff
@@ -1138,9 +1155,8 @@ func (c *statusContext) processApplicationRelations(
 			related[relationName] = append(related[relationName], ep.ApplicationName)
 		}
 	}
-	for relationName, applicationNames := range related {
-		sn := set.NewStrings(applicationNames...)
-		related[relationName] = sn.SortedValues()
+	for _, applicationNames := range related {
```

Comment:
This was de-duplicating them, I think we still need that if we can't guarantee that from the query.